Jenkinsfile

```diff
@@ -102,7 +102,7 @@ pipeline {
       }
       post {
         always {
-          publishCoverage adapters: [cobertura('coverage/storybook/cobertura-coverage.xml')], checksName: 'coverage', sourceFileResolver: sourceFiles('NEVER_STORE')
+          recordCoverage name: 'coverage', sourceCodeRetention: 'NEVER', tools: [[parser: 'COBERTURA', pattern: 'coverage/storybook/cobertura-coverage.xml']]
```

Comment:
how does sourcecode retention of never work when there's no workspaces kept around? will it just report totals or something then?

....and I see it was essentially that in the past, so its probably fine.

Reply:
There shouldn't be a change to the previous layout, given files weren't stored before, either.